devel.yaml.jinja

```diff
@@ -48,7 +48,9 @@ services:
       - ./odoo/auto:/opt/odoo/auto:rw,z
     depends_on:
       - cdnjs_cloudflare_proxy
+      {% if postgres_version -%}
```

Comment:
Our code standard here for these kind of blocks is `{%- ... %}`.

Whitespace is weird in jinja and important in yaml.

Please follow it wherever possible (and here is possible). Same for all others.

Reply:
Ok. I actually did this following your suggestion in https://github.com/Tecnativa/doodba-copier-template/commit/abad70316b2bf8d924784615926c87fff6d0f1c2
I guess it doesn't apply the same here, as we are modifying a yaml array, and not adding the service, as before, right?

Reply:
Shouldn't we put the same condition on 'pgweb' dependency and on the 'db' under volumes as well?

Reply:
>  I guess it doesn't apply the same here, as we are modifying a yaml array, and not adding the service, as before, right?

Well, your code is not wrong here, but it was wrong there. :yum: 

the `-` is telling jinja to remove all whitespace from (or to) that boundary. Wherever possible, the code standard here is to remove previous whitespace, and here it is possible. In the other code you link, it was not possible (it removed one extra line).



> Shouldn't we put the same condition on 'pgweb' dependency and on the 'db' under volumes as well?

Yes, on every service that depends on `db` actually

Reply:
In that case, IMOH below choice should be removed from copier question:

`I will use an external PostgreSQL server: null`

And also it is not clear in docs that how to change DB parameters when deploying in production. It will be good to add some help for that.

Reply:
> Nope. Only on prod.yaml.

In that case, if the  db service should always exist in devel and test, maybe we should end up with a different copier answer or mechanism to set up the case when we want an external DB on prod. If we set this var to `null`, we can't know which version to use in devel and test, right?

Maybe add another question with "Whether or not ti use external server?", that would override the db configuration only in prod.

Reply:
> Maybe add another question with "Whether or not ti use external server?", that would override the db configuration only in prod.
Or maybe split db questions for prod and dev/test. For instance, there are separate questions for production (domains_prod) and test domains (domains_test).

Reply:
Yes, we have to rethink the UX with this question.